lambdatimes/src/components/Content/Content.js

```diff
@@ -31,26 +47,27 @@ export default class Content extends Component {
       
       Notice that we're passing this function to our <Cards /> component below.
       This function returns an array of cards, so we can just pass it down as such.
-
+  
       Your algorithim for the logic here is as follows: 
         - if the selected tab is 'all' it should return all 
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
+
+    if (this.state.selected == "all") {
+      return this.state.cards;
+    } else {
+      return this.state.cards.filter(card => card.tab === this.state.selected);
+    }
   };
 
   render() {
     return (
-      <div className="content-container">
-        {/* 
-          Add 2 props to the Tabs component, 
-          `selectedTab` that includes the currently selected tab
-          and `selectTabHandler` that includes the function to change the selected tab
-        */}
-        <Tabs tabs={this.state.tabs} />
+      <StyledContent>
+        <Tabs tabs={this.state.tabs} selectedTab={this.state.selected} selectTabHandler={this.changeSelected} />
+        <Carousel selectedImage={this.props.selectedImage} />
         <Cards cards={this.filterCards()} />
```

Comment:
Nice work on the filter function. This was a tricky task with a new approach.

Reply:
Let me know if you have specific questions relating to the filterCards invocation on this line.